### PR TITLE
Rewrite "Finite Markov Chains"

### PR DESCRIPTION
**Pull request by arnavs**
#160. Nothing major. Just mainly: 

1. Moved away from QuantEcon.jl's `DiscreteRV` to Bernoulli or Categorial from Distribution.
2. Made some style guide/legibility changes.

### CODE REVIEW

**Comment by Nosferican** [comment]
Not sure if we want to tackle this at the moment, but in my experience some of the images should be reduced from 100% size. If that is the case in your end, you could do so now or leave it to the images pass. Other than that, it seems fine.

**Comment by arnavs** [comment]
Sure, and we can add that to the style guide as well (@jlperla). Something like "make sure your images take up no more than X% of screen width."

**Comment by arnavs** [comment]
Sounds good @jlperla. Going to merge this then and we can revisit towards the end.

**Comment by jlperla** [comment]
Yeah, this doesn't seem to use anything too advanced that we didn't introduce in the intro slides.

File: rst_files/finite_markov.rst [hunk 1]
```diff
@@ -1428,28 +1268,21 @@ compare it to the stationary probability.
...
 
+    α = 0.1 # probability of getting hired
+    β = 0.1 # probability of getting fired
+    N = 10000
+    p_bar = β / (α + β) # steady-state probabilities
+    P = [1 - α   α
+         β   1 - β] # stochastic matrix
     mc = MarkovChain(P)
+    labels = ["start unemployed", "start employed"]
+    y_vals = Array{Vector}(undef, 2) # sample paths holder
```
Comment (arnavs) [inline rst_files/finite_markov.rst]: Should definitely replace this with a matrix if possible.

**Pull request merged**